tests/test_constants.py

```diff
@@ -21,39 +37,419 @@ def test_constants(self):
         """
         self.maxDiff = None
 
-        self.assertEqual(constants.CMD_CURRENT_APP, "CURRENT_APP=$(dumpsys window windows | grep mCurrentFocus) && CURRENT_APP=${CURRENT_APP#*{* * } && CURRENT_APP=${CURRENT_APP%%/*} && echo $CURRENT_APP")
+        # CMD_CURRENT_APP
+        self.assertEqual(constants.CMD_CURRENT_APP, "CURRENT_APP=$(dumpsys window windows | grep -E 'mCurrentFocus|mFocusedApp') && CURRENT_APP=${CURRENT_APP#*ActivityRecord{* * } && CURRENT_APP=${CURRENT_APP#*{* * } && CURRENT_APP=${CURRENT_APP%%/*} && CURRENT_APP=${CURRENT_APP%\}*} && echo $CURRENT_APP")
```

Comment:
Copy & pasting your explanation from [here](https://github.com/JeffLIrion/python-androidtv/pull/233#discussion_r571486365) (for reference):

1. `${CURRENT_APP#*ActivityRecord{* * }`
Attempt to extract application activity from `mResumedActivity` or `mFocusedApp`
2. `${CURRENT_APP#*{* * }`
Fallback to extract application activity from `mCurrentFocus`, in case `mFocusedApp` is not defined
3. `${CURRENT_APP%%/*}`
Remove extraneous activity details for extracted application, leaving only identifier
4. `${CURRENT_APP%\\}*}`
Delete trailing curly brace from identifier of extracted application, in case application has no activity